openmp/libomptarget/DeviceRTL/src/Parallelism.cpp

```diff
@@ -83,6 +83,64 @@ uint32_t determineNumberOfThreads(int32_t NumThreadsClause) {
 
 extern "C" {
 
+[[clang::always_inline]] void
+__kmpc_parallel_spmd(IdentTy *ident, int32_t, int32_t if_expr,
```

Comment:
Can we please not add unused/unnamed arguments?

Reply:
My bad, got rid of them.